### PR TITLE
Homepage2

### PR DESCRIPTION
**Pull request by ced37**

### CODE REVIEW

**Review by floriane-cheneveau — DISMISSED** [review]

File: src/View/Home/nav.html.twig [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{% extends 'layout.html.twig' %}
+{% block nav %}
+    <nav id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <div class="overlay-content">
+        <a href="#">Accueil</a>
```
Comment (arnaud-blanc) [inline src/View/Home/nav.html.twig]:
```suggestion
        <a href="/">Accueil</a>
```

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -5,17 +5,24 @@
...
+        {% include 'Home/nav.html.twig' %}
+        {% endblock %}
+    </header>
     {% block content %}{% endblock %}
     {% block javascript %}{% endblock %}
 </body>
-</html>
-
 
+{# Ces codes javascript qui suivent sont temporaires, ils seront remanié puis ajouté dans un block javascript.#}
```
Comment (arnaud-blanc) [inline src/View/layout.html.twig]: These following javascript codes are temporary, they will be refactored and added in a javascript file

File: public/assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+body {
```
Comment (arnaud-blanc) [inline public/assets/css/style.css:1]: These styles belong to the curtain nav bar.

File: public/assets/javascript/scriptNav.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+function openNav() {
```
Comment (arnaud-blanc) [inline public/assets/javascript/scriptNav.js:1]: Nav bar and top button script.

File: src/View/Home/index.html.twig [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
-    <h1>Hello Wilder !</h1>
+    
+    <h1>Clevertricks</h1>
+        <p>Trucs et Astuces pour Développeurs</p>
+
+        {#<label for="site-search"></label>
+        <input type="search" id="site-search" name="q"
+        placeholder="Rechercher sur le site…">
+
+        <button>Go!</button></br>#}
```
Comment (arnaud-blanc) [inline src/View/Home/index.html.twig]:
```suggestion
```

**Review by arnaud-blanc — CHANGES_REQUESTED** [review]

**Review by arnaud-blanc — APPROVED** [review]

File: src/View/layout.html.twig [hunk 6]
```diff
@@ -5,17 +5,24 @@
...
     <link rel="stylesheet" href="/assets/css/style.css">
     {% block stylesheet %}{% endblock %}
     <link rel="icon" href="/assets/images/favicon.png">
 </head>
 
 <body>
+    <header>
+        {% block nav %}
+        {% include 'Home/nav.html.twig' %}
+        {% endblock %}
```
Comment (RomainClair) [inline src/View/layout.html.twig:19]: You'd better have the navbar HTML part here if you want your navbar to be on every page.

**Review by RomainClair — CHANGES_REQUESTED** [review]
It's really strange to include a twig that extends a layout in that layout.
You should use include or use template inheritance but not both for the same purpose.

**Pull request closed without merging**